controller/cmd/identity/main.go

```diff
@@ -117,12 +101,19 @@ func Main(args []string) {
 	issuerError := make(chan error)
 
 	//
-	// Create and start FS creds watcher
+	// Create and start FS creds watchers
 	//
-	watcher := tls.NewFsCredsWatcher(*issuerPath, issuerEvent, issuerError)
+	issuerWatcher := tls.NewFsCredsWatcher(*issuerPath, issuerEvent, issuerError)
+	go func() {
+		if err := issuerWatcher.StartWatching(ctx); err != nil {
+			log.Fatalf("Failed to start issuer watcher: %s", err)
+		}
+	}()
+
+	trustRootWatcher := tls.NewFsCredsWatcher(k8s.MountPathTrustRootsBase, issuerEvent, issuerError)
```

Comment:
> The identity controller is updated to mount this new configmap as a volume and to watch it for changes, just as it does for the issuer credentials.

This is not generally safe. The identity controller's proxy won't be able to observe these changes, so there's no functional benefit to watching the CA configmap--we'll always need to restart control plane components when this configmap changes.

It might be okay practically as long as users don't change the configmap in a way that conflicts with running proxies; but I think it might be best to just load it at startup and not watch it.